internal/configtxgen/genesisconfig/config.go

```diff
@@ -66,6 +69,17 @@ const (
 	AdminRoleAdminPrincipal = "Role.ADMIN"
 )
 
+// configCache stores marshalled bytes of config structures that produced from
+// EnhancedExactUnmarshal. Cache key is the path of the configuration file that was used.
+type configCache struct {
+	mux   sync.Mutex
```

Comment:
A `mux` is a multiplexor, not a mutex. It's a bit of a confusing name.

Reply:
done.